src/lib/file.js

```diff
@@ -48,25 +48,29 @@ export const parseA8MWFile = (file) => {
         .loadAsync(e.target.result)
         .then(parseConfigFile)
         .then(extractEntryData)
-        .then(({ text, config }) => {
+        .then(async ({ text, config, contents }) => {
+          const imagesFolder = contents.folder('images');
+          const imageFiles = imagesFolder 
+            ? Object.entries(contents.files)
+                .filter(([path]) => path.startsWith('images/') && path !== 'images/')
+                .reduce((acc, [path, file]) => {
+                  acc.files[path] = file;
+                  return acc;
+                }, { files: {} })
+            : { files: {} };
+          
           resolve({
             text,
-            config: asConfigData(config),
+            config: {
+              ...asConfigData(config),
+            },
+            imagesFolder: imageFiles
           });
         })
-        .catch((e) => {
-          reject(e);
-        });
+        .catch(reject);
     };
 
-    reader.onerror = (e) => {
-      console.error(e);
-      reject(e);
-    };
-
-    reader.onabort = (e) => {
-      console.log('onabort', e);
-    };
```

Comment:
I'd like to know the thought you don't want to log here and remove the integration with onabort callback.